google-cloud-pubsub/lib/google/cloud/pubsub/project.rb

```diff
@@ -331,32 +296,14 @@ def publish topic_name, data = nil, attributes = {}
         #                          deadline: 120,
         #                          endpoint: "https://example.com/push"
         #
-        # @example With `autocreate`:
-        #   require "google/cloud/pubsub"
-        #
-        #   pubsub = Google::Cloud::Pubsub.new
-        #
-        #   sub = pubsub.subscribe "new-topic", "new-topic-sub",
-        #                          autocreate: true
-        #
         def subscribe topic_name, subscription_name, deadline: nil,
-                      retain_acked: false, retention: nil, endpoint: nil,
-                      autocreate: nil
+                      retain_acked: false, retention: nil, endpoint: nil
           ensure_service!
           options = { deadline: deadline, retain_acked: retain_acked,
                       retention: retention, endpoint: endpoint }
           grpc = service.create_subscription topic_name,
                                              subscription_name, options
           Subscription.from_grpc grpc, service
-        rescue Google::Cloud::NotFoundError => e
```

Comment:
The documentation for this method does not state that `nil` will be returned when the resource does not exist. This is because `NOT_FOUND` can be raised for a missing Topic resource or a missing Subscription resource. The users have to check the error to see for sure.